packages/codec/sources/base64.move

```diff
@@ -35,17 +49,20 @@ public fun encode(mut bytes: vector<u8>): String {
 public fun decode(str: String): vector<u8> {
     let keys = KEYS;
     let mut res = vector[];
+    let mut len = str.length();
```

Comment:
I'd probably assign `bytes.length()` to the len var, to keep it easier to follow.

Reply:
Nice catch. Patched.